misc/packaging/acrn-hypervisor.postinst

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+#* Copyright (c) 2020 Intel Corporation
+# postinst script for acrn-hypervisor
+set -e
+
+
+echo "please choose <scenario> ,<board> ,<disk type>"
+
+echo "Senario is ->
+        1. industry
+        2. hybrid
+        3. logical_partition"
+read num
+
+if [[ $num -eq 1 ]]
+then
+        echo "Senario is industry"
+        SCENARIO="industry"
+elif [[ $num -eq 2 ]]
+then
+        echo "Senario is hybrid"
+        SCENARIO="hybrid"
+
+elif [[ $num -eq 3 ]]
+then
+        echo "Senario is logical_partition"
+        SCENARIO="logical_partition"
+fi
+
+echo "Board is ->
```

Comment:
These are the only two supported boards?  What about all the other supported hardware we list?

Reply:
this version will include two boards